### PR TITLE
Day08

### PR DESCRIPTION
**Pull request by thecardkid**
Lord Almighty.

### CODE REVIEW

File: day08/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Day08: Enhancing your indexer and crawler
```
Comment (ssingal05) [inline day08/README.md]: There are less method definitions here in the README than there are TODOs in the code below. Either add methods to this README or delete methods below.

Comment (thecardkid) [inline day08/README.md]: Fixed

Comment (ssingal05) [inline day08/README.md]: you need "getCount"

File: day08/src/WikiFetcher.java [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+		sleepIfNeeded();
+		
+		// download and parse the document
+		Connection conn = Jsoup.connect(url);
+		Document doc = conn.get();
+		
+		// select the content text and pull out the paragraphs.
+		Element content = doc.getElementById("mw-content-text");
+		
+		// TODO: avoid selecting paragraphs from sidebars and boxouts
```
Comment (ssingal05) [inline day08/src/WikiFetcher.java]: Remove Allen TODO

File: day08/src/WikiFetcher.java [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+		// assemble the file name
+		String slash = File.separator;
+		String filename = "resources" + slash + realURL.getHost() + realURL.getPath();
+
+		// read the file
+		InputStream stream = WikiFetcher.class.getClassLoader().getResourceAsStream(filename);
+		Document doc = Jsoup.parse(stream, "UTF-8", filename);
+
+		// parse the contents of the file
+		// TODO: factor out the following repeated code
```
Comment (ssingal05) [inline day08/src/WikiFetcher.java]: remove Allen TODO

**Pull request merged**